pkgs/by-name/ma/mautrix-slack/package.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  olm,
+  nix-update-script,
+  testers,
+  mautrix-slack,
+  # This option enables the use of an experimental pure-Go implementation of
+  # the Olm protocol instead of libolm for end-to-end encryption. Using goolm
+  # is not recommended by the mautrix developers, but they are interested in
+  # people trying it out in non-production-critical environments and reporting
+  # any issues they run into.
+  withGoolm ? false,
+}:
+let
+  version = "0.1.4";
+in
+buildGoModule {
+  pname = "mautrix-slack";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "mautrix";
+    repo = "slack";
+    rev = "v${version}";
+    hash = "sha256-MDbWvbEY8+CrUL1SnjdJ4SqyOH/5gPsEQkLnTHyJdOo=";
+  };
+
+  vendorHash = "sha256-8U6ifMLRF7PJyG3hWKgBtj/noO/eCXXD60aeB4p2W54=";
+
+  buildInputs = lib.optional (!withGoolm) olm;
+  tags = lib.optional withGoolm "goolm";
+
+  doCheck = false;
```

Comment:
Please add a comment explaining why checks are disabled

Reply:
I've checked again, and they seem to be working, so I re-enabled them.